contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveMetadataProvider.java

```diff
@@ -355,7 +355,7 @@ public LogicalInputSplit(Collection<? extends InputSplit> inputSplits, Partition
       this.partition = partition;
     }
 
-    public Collection<InputSplit> getInputSplits() {
+    public List<InputSplit> getInputSplits() {
```

Comment:
Why `Collection`->`List`? Where order comes from?

Reply:
It is not a question whether it is known to be a `List` or an `ArrayList` (it is probably known that it is an `ArrayList`). `List` provides certain behavior (allows duplicate and provides insertion order guarantee). Do you need any of those?

Reply:
Small, but right remark here. Thanks. I have returned this to `Collection`. There is no need to use `List`.

I think `Collection` is used because of a `Map#values()` returns it in some places. 
From the code it looks like it is enough (at least now) the API of `Collection` for `logicalInputSplit`. 
Also I found other places with `Collection<FileSplit> logicalInputSplit` and `List<LogicalInputSplit> getInputSplits()`. Possibly they should be leaded to the common style.